#!/usr/bin/python3
def raise_exception_msg(msg=""):
    exception = NameError(msg)
    raise exception
